### DISCUSSION TITLE
use-multiple-aws-accounts-using-aws-cli

### DISCUSSION THREAD

**Discussion by giscus** [thomasnotfound/how.wtf#232] [Blog Posts]
# use-multiple-aws-accounts-using-aws-cli

How to use multiple AWS accounts using the AWS CLI

https://how.wtf/use-multiple-aws-accounts-using-aws-cli.html

**Comment by yoonsoo-park** [comment]
This is super helpful!

**Reply by thomasgtaylor** [reply]
Thank you for your kind words!
